fixtures/onwards.mocks.ts

```diff
@@ -216,3 +216,12 @@ export const withLongDescription: OnwardsType = {
 	ophanComponentName: 'more-on-this-story',
 	pillar: Pillar.News,
 };
+
+export const withLongDescriptionAndLink: OnwardsType = {
```

Comment:
Maybe just call this `withLink`? Simpler. You can perhaps make the text shorter to be accurate!

Reply:
Good Shout